app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -3,19 +3,41 @@
 module Api
   module V1
     class LobbiesController < AuthenticatedController
-      before_action :require_authentication, except: [:from_code, :join]
+      before_action :require_authentication, except: [:from_code, :join, :answer]
+      before_action :set_lobby, only: [:join, :start, :answer, :show, :update, :destroy]
       before_action :require_authorisation, only: [:show, :update, :destroy]
```

Comment:
`start` should also be authorised. You don't want users to be able to start someone else's lobby 😄